charts/openfga/values.schema.json

```diff
@@ -506,6 +506,13 @@
                                 "type": "string",
                                 "minItems": 1
                             }
+                        },
+                        "keysSecret": {
+                          "type": [
+                              "string",
+                              "null"
+                          ],
+                          "description": "the secret name where to get the preshared keys, it expects a key named 'presharedKeys' to exist in the secret containing a comma-separated list of keys"
```

Comment:
I would clarify in the description that if `authn.method=preshared` we need either this or the other property, it would be good to update the description of `preshared.keys` as well to reflect that there are two ways to pass them with this change.